challenge.rb

```diff
@@ -45,7 +53,18 @@ def okay_to_drink? age, drinking_age
 #   "Fix the AC whenever you have the chance. It's cool."
 
 def ac_monitor current_temp, ac_working, desired_temp
-  # Replace this comment with your code.
+  if current_temp > desired_temp && ac_working == true
+    return "Turn on the AC."
+  end
+  if current_temp <= desired_temp && ac_working == true
+    return "Just right!"
+  end
+  if current_temp > desired_temp && ac_working == false
+    return "Fix the AC now! It's hot!"
+  end
+  if current_temp <= desired_temp && ac_working == false
+    return "Fix the AC whenever you have the chance. It's cool."
+  end
```

Comment:
See previous comment about `== true`. `x == false` is similarly redundant, since it will always be equal to `!x`. 

This set of conditions could also have been written a bit more tersely. Ultimately, we only need to perform two logical tests to break apart all four possible cases: (1) is the AC working, and (2) is it too hot. Imagine if we wrote the following:
```
if ac_working
  if current_temp > desired_temp
    "Turn on the AC"
  else
    "Just right."
  end
else 
  if current_temp > desired_temp
    "Fix the AC now! It's hot!"
  else 
    "Fix the AC whenever you have the chance. It's cool."
  end
end
```